skbio/util/_testing.py

```diff
@@ -8,6 +8,53 @@
 
 import os
 import inspect
+from nose import core
+from nose.tools import nottest
+from future.utils import PY3
+
+
+@nottest
+class TestRunner(object):
+    """Simple wrapper class around nosetests functionality.
+
+    Parameters
+    ----------
+    filename : __file__ attribute passed in from the caller. This tells the
+        tester where to start looking for tests.
+
+    Attributes
```

Comment:
Not sure where to put this comment. In response to your question about documenting private member variables, we generally leave these out and only document things that are public (so this section can be removed).